server/command.go

```diff
@@ -246,3 +247,27 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 	p.postCommandResponse(args, "Unknown action %v", action)
 	return &model.CommandResponse{}, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	welcomebot := model.NewAutocompleteData("welcomebot", "[command]",
+		"Available commands: preview, help, list, set_channel_welcome, get_channel_welcome, delete_channel_welcome")
+
+	preview := model.NewAutocompleteData("preview", "[team-name]", "Preview the welcome message for the given team name")
+	preview.AddTextArgument("Team Name to preview welcome message", "[team-name]", "")
```

Comment:
One small nitpick and this PR looks good!

```suggestion
	preview.AddTextArgument("Team name to preview welcome message", "[team-name]", "")
```

Reply:
Updated with changes https://github.com/mattermost/mattermost-plugin-welcomebot/pull/69/commits/c030af75579b6538197badf8ff65273bcf169de8